### ISSUE TITLE
Including p value outputs for the coefficients  in the regression and classification models 

### ISSUE THREAD

**Issue by Blesst88** [biolab/orange3#5845]
**What's your use case?**
The current statistical analysis does not provide the statistically significance of the each feature used in the model 
This limits the value of the analysis as we could not determine if the feature contributes significantly to the model 

Currently  have to use other statistical packages to compute the p value for each co-efficient 

**What's your proposed solution?**
It would be better if this could be included in the summary reports from the model and shown in data table 

**Are there any alternative solutions?**
Use other packages such SPSS or JASP

**Comment by janezd** [comment]
We have already touched this topic at least twice.

Once, it was about Correlations (and probably about some variations of VizRank, which essentially compute p-values). In these cases, we decided not to show p-values because people would tend to use them as *statistical significance* and reject the related "null hypotheses", ignoring the fact that they'd have to be corrected for multiple hypothesis testing.

A more general consideration here is that Orange is a tool for explorative data analysis, which is about creating hypotheses from data. Even if you don't use tools that systematically search for hypotheses (like VizRank or Correlations), the user would certainly draw, for instance, a number of scatter plots, pick the one that (s)he likes and then compute the significance of correlation.

From another angle, an internal discussion about this widget (related to a similar suggestion, #5722) even led to a suggestion to remove regression lines altogether (or, perhaps, keep those that treat the two variables as independent, because this is what scatter plot naturally shows). We said the linear regression should be a separate widget with a reacher set of functions (see #5733). Adding them to scatter plot would simply overload the widget.

I would thus vote against adding this feature (however trivial it would be to add it). Also, personally, I strongly believe we should stop the NHST ritual and start relying on Bayesian statistics.
